### ISSUE TITLE
link to fft(`Double64s`) works on Julia v1.3.0-DEV, not on v1.1.1

### ISSUE THREAD

**Issue by JeffreySarnoff** [JuliaApproximation/FastTransforms.jl#68]
https://github.com/JuliaMath/AbstractFFTs.jl/issues/28
(not sure which would be the better place to post this issue)
```
julia> VERSION # v"1.3.0-DEV.263"
julia> using DoubleFloats, FastTransforms, FFTW

julia> let df64s = rand(Double64, 2^10)
         ComplexDF32( sum(df64s .- ifft(fft(df64s))) )
       end
3.9988234e-30 + 0.0im
```

```
julia> VERSION # v"1.1.1"
julia> using DoubleFloats, FastTransforms, FFTW

julia> let df64s = rand(Double64, 2^10)
         ComplexDF32( sum(df64s .- ifft(fft(df64s))) )
       end
ERROR: type DoubleFloat{Float64} not supported
Stacktrace:
 [1] error(::String) at .\error.jl:33
 [2] _fftfloat(::Type{DoubleFloat{Float64}}) at C:\Users\jas\.julia\packages\AbstractFFTs\PUqOK\src\definitions.jl:22
 [3] _fftfloat(::DoubleFloat{Float64}) at C:\Users\jas\.julia\packages\AbstractFFTs\PUqOK\src\definitions.jl:23
 [4] fftfloat(::DoubleFloat{Float64}) at C:\Users\jas\.julia\packages\AbstractFFTs\PUqOK\src\definitions.jl:18
 [5] complexfloat(::Array{DoubleFloat{Float64},1}) at C:\Users\jas\.julia\packages\AbstractFFTs\PUqOK\src\definitions.jl:31
 [6] fft(::Array{DoubleFloat{Float64},1}, ::UnitRange{Int64}) at C:\Users\jas\.julia\packages\AbstractFFTs\PUqOK\src\definitions.jl:198 (repeats 2 times)
 [7] top-level scope at REPL[3]:2
```

**Comment by giordano** [comment]
As has been noted in https://github.com/JuliaMath/AbstractFFTs.jl/issues/28 this feature not in a released version of this package.

**Comment by dlfivefifty** [comment]
Should we tag a new release?

**Comment by giordano** [comment]
The last one was in November, so maybe it would be nice.  But probably you want to update the docs before doing the release, I couldn't find a reference to the FFT feature anywhere (am I missing something?)

**Comment by dlfivefifty** [comment]
I have time to tag, but no time to work on docs. Feel free to make a PR to the docs.

**Comment by JeffreySarnoff** [comment]
fixed in v0.5, thank you
